about.html

```diff
@@ -90,6 +90,7 @@ <h3>2021</h3>
             </article>
           </div>
         </section>
+      </section>
```

Comment:
## GOOD JOB SO FAR

Let me answer your question. Yes, you have time until the deadline. I think about 6-10 hours ⏲. But I highly recommend the issues I give you asap 💡

Reply:
Thanks for saying that. 🙌